internal/codespaces/ssh.go

```diff
@@ -95,9 +83,56 @@ func newSSHCommand(ctx context.Context, port int, dst string, cmdArgs []string)
 	return cmd, connArgs, nil
 }
 
-// parseSSHArgs parses SSH arguments into two distinct slices of flags and command.
-// It returns an error if a unary flag is provided without an argument.
 func parseSSHArgs(args []string) (cmdArgs, command []string, err error) {
+	return parseArgs(args, "bcDeFIiLlmOopRSWw")
+}
+
+// newSCPCommand populates an exec.Cmd to run an scp command for the files specified in cmdArgs.
+// cmdArgs is parsed such that scp flags and the files to copy are separated by a "--" in the command.
+// For example: scp -F ./config -- local/file remote:file
+func newSCPCommand(ctx context.Context, port int, dst string, cmdArgs []string) (*exec.Cmd, error) {
+	// Beware: invalid syntax causes scp to exit 1 with
+	// no error message, so don't let that happen.
```

Comment:
Can we move this command down to the `exec.CommandContext` line?

Reply:
Done